projects/ui-components/lf-metadata/lf-field-template-container/lf-field-template-container.component.ts

```diff
@@ -148,6 +175,36 @@ export class LfFieldTemplateContainerComponent extends LfFieldContainerDirective
     });
   };
 
+  /** @internal */
+  get isDropdownError(): boolean {
+    return this.dropdownLabelState === DropDownState.HAS_ERROR;
```

Comment:
dropdownState instead of dropdownLabelState. I keep thinking this is the "No template assigned" label